### PR TITLE
Fix navigation icon bullet accessibility

### LINKED ISSUE #3955: Calendar visibility toggle (entry itself) is not focusable via keyboard

### LINKED ISSUE #3966: Color indicator circle of disabled calendars is almost invisible

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
use CheckboxBlankCircle as icon within the Action for icon bullet instead of AppNavigationIconBullet and AppNavigationDisabledCalendarIconBullet.

AppNavigationDisabledCalendarIconBullet is not needed anymore.

fixes #3955 
fixes #3966

- [x] Requires https://github.com/nextcloud/nextcloud-vue/pull/2506

### CODE REVIEW

**Comment by GretaD** [comment]
is this ready for review? also, how to test it?

**Comment by JuliaKirschenheuter** [comment]
@nimishavijay 
i have done some small UI changes (only calendar app, only sidebar), please approve it if it is ok.

now (with focus on circle):
image

before (without focus on circle): 
image

**Comment by JuliaKirschenheuter** [comment]
fixes #3955

**Review by nimishavijay — APPROVED** [review]
Looks great design wise! 🚀

**Comment by ChristophWurst** [comment]
this requires changes in nc/vue, doesn't it?

File: src/components/AppNavigation/CalendarList/CalendarListItem.vue [hunk 1]
```diff
@@ -26,15 +26,18 @@
...
-		<AppNavigationDisabledCalendarIconBullet
-			v-if="!calendar.enabled"
-			slot="icon"
-			@click.prevent.stop="toggleEnabled" />
+		<template slot="icon">
+			<Actions>
+				<ActionButton @click.prevent.stop="toggleEnabled">
+					<template #icon>
+						<CheckboxBlankCircle
+							:title="isCalendarEnabled ? $t('calendar', 'Disable calendar') : $t('calendar', 'Enable calendar') "
```
Comment (ChristophWurst) [inline src/components/AppNavigation/CalendarList/CalendarListItem.vue:34]:
```suggestion
							:title="calendar.enabled ? $t('calendar', 'Disable calendar') : $t('calendar', 'Enable calendar') "
```

that is simple enough and doesn't require an additional computed property. same is used two lines below

File: src/components/AppNavigation/CalendarList/CalendarListItem.vue [hunk 2]
```diff
@@ -195,6 +196,7 @@ import LinkVariant from 'vue-material-design-icons/LinkVariant.vue'
 import Pencil from 'vue-material-design-icons/Pencil.vue'
 import ShareVariant from 'vue-material-design-icons/ShareVariant.vue'
 import Undo from 'vue-material-design-icons/Undo.vue'
+import CheckboxBlankCircle from 'vue-material-design-icons/CheckboxBlankCircle.vue'
```
Comment (ChristophWurst) [inline src/components/AppNavigation/CalendarList/CalendarListItem.vue:199]: move up so the icons are sorted alphabetically

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

**Pull request merged**